Kudu.Core/Jobs/ContinuousJobsManager.cs

```diff
@@ -57,6 +62,39 @@ public void EnableJob(string jobName)
             continuousJobRunner.EnableJob();
         }
 
+        public async Task<HttpResponseMessage> RequestPassthrough(string jobName, string path, HttpRequestMessage request)
```

Comment:
Should this belong on the ContinuousJobRunner class? Also, please add a unit test for this method. I see there is a ContinuousJobRunnerFacts unit test class already. Otherwise, we may need a ContinuousJobManagerFacts class.